packages/backend/src/core/TutorialService.ts

```diff
@@ -0,0 +1,17 @@
+import { HomeTutorialEntry } from '@explorer/frontend'
+import fs from 'fs'
+
+export class TutorialService {
+  getTutorials(): HomeTutorialEntry[] {
+    const files = fs.readdirSync('src/content/tutorials')
```

Comment:
Will the tutorials always be platform neutral? So the same for e.g. dydx and apex?

Reply:
Well I would say so, if not we would filter by some prefix e.g. apex-example.md